src/components/ErrorComponent.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const ErrorComponent = () => {
+  return (
+    <div>ErrorComponent</div>
```

Comment:
here can print the reason of the error, something like errorText.

Reply:
sure, noted.